### ISSUE TITLE
ami: coredump directory not configured

### ISSUE THREAD

**Issue by avikivity** [scylladb/scylla#1388]
*Installation details*
Scylla version (or git commit hash): 1.2
OS (RHEL/CentOS/Ubuntu/AWS AMI): AMI

`/var/lib/systemd/coredump` should be bind-mounted to the data directory, but it isn't.

**Comment by slivne** [comment]
scylla_coredump_setup has an option to 
```
        rm -rf /var/lib/systemd/coredump
        ln -sf /var/lib/scylla/coredump /var/lib/systemd/coredump
```

Is this what you want added or something else ? (in this case we simply need to invoke scylla_coredump_setup with the ```--dump-to-raiddir```)


 - if its something else - should we modify the sctkka_coredump_setup so it will apply not only to the ami.

**Comment by slivne** [comment]
@avikivity  please respond so we will know if its only calling the script

**Comment by slivne** [comment]
@avikivity ping

**Comment by syuu1228** [comment]
Should be fixed in the patch recently sent "[PATCH scylla] setup coredump on first startup"

**Comment by slivne** [comment]
fixed by scylla-ami@14c1666423fb246c8a54cdf7f4bd40db8c04be3b
